CHANGELOG.rst

```diff
@@ -3,6 +3,32 @@ Release Notes
 
 List of the most important changes for each release.
 
+0.11.1
+------
+
+Added
+~~~~~
+
+- Support for RTL EPubs
+
+Changed or fixed
+~~~~~~~~~~~~~~~~
+
+- Fix: Fullscreen renderer mode now works in Chrome 71
+- Fix: Account sign up now works when guest access is disabled
+- Fix: Navigating in and out of exercise detail views
+- Fix: Misleading exam submission modal text
+- Fix: Incredibly slow loading when browsing content tree in exam creation
+- Fix: Loading errors for unavailable content in coach reports
+- Fix: Better content import error catching
+- Added: Support for Python 3.7
+- Added: Command to restore availability of content after bad upgrade
```

Comment:
these 'fix' and 'added' prefixes are weird: they seem redundant/contradictory with the 'added'/'fixed' headers